src/back-end/lib/routers/auth.ts

```diff
@@ -136,11 +137,11 @@ async function makeRouter(connection: Connection): Promise<Router<any, any, any,
         }
       })
     },
-    // /auth/createsession is for testing only, so only exists outside of production
+    // /auth/createsessiongov is for testing only, so only exists outside of production
     // @ts-ignore
     ...(process.env.NODE_ENV === 'development' ? [{
       method: ServerHttpMethod.Get,
-      path: '/auth/createsession',
+      path: '/auth/createsessiongov',
```

Comment:
I couldn't think of a way to use the same endpoint to authenticate different users. I thought of env variables, but [Cypress doesn't seem to be able to change OS-level variables ](https://docs.cypress.io/guides/guides/environment-variables#:~:text=Difference%20between%20OS%2Dlevel%20and,from%20OS%2Dlevel%20environment%20variables.)

Reply:
That's fine, your solution works